src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -274,6 +276,31 @@ CLParser::Usage(const std::string& msg)
              "not be 0 for sequence models while using asynchronous mode.",
              18)
       << std::endl;
+  std::cerr
+      << FormatMessage(
+             "--periodic-concurrency-range <start:end:step>: Determines the "
+             "range of concurrency levels in the similar manner as the "
+             "--concurrency-range. The perf_analyzer will start from the "
+             "concurrency level of 'start' and go until it reaches 'end' with "
+             "a stride of 'step'. Unlike --concurrency-range, the user can "
+             "specify *when* to periodically increase the concurrency level "
+             "using the --request-period option. The concurrency level will "
+             "periodically increase for every n-th response specified by "
+             "--request-period. Since this disables stability check in "
```

Comment:
```suggestion
             "--request-period=<n>. Since this disables stability check in "
```

Reply:
Instead of adding that parameter information in this paragraph, I will add it in the `--request-period` section.